dbt_copilot_helper/templates/addons/env/opensearch.yml

```diff
@@ -43,6 +44,7 @@ Mappings:
 
 Conditions:
   {{ addon_config.prefix }}EnableHA: !Not [!Equals [!FindInMap [{{ addon_config.prefix }}EnvironmentConfigMap, !Ref Env, InstanceCount], 1]]
+  CreateProdSubFilter: !Or [!Equals [!Ref Env, prod], !Equals [!Ref Env, production], !Equals [!Ref Env, PROD], !Equals [!Ref Env, PRODUCTION]]
```

Comment:
Does this condition need to be prefixed with `{{ addon_config.prefix }}`?

Reply:
No, as its important to be unique to each env only.  Within an env the condition is valid for multiple resources.

Reply:
actually I will put it in as there is no harm in having it.